### DISCUSSION TITLE
hide default tempo and dynamic in alphatex

### DISCUSSION THREAD

**Discussion by sugizo** [CoderLine/alphaTab#455] [Q&A] [answered]
# Question 
how to hide default tempo and dynamic in alphatex ?

**tried** 
**value** = none, false, disable, disabled, hide, hidden, 0
for both tempo and dynamic not work
```
. 
\ts 0 0 \tempo 0
0.1{dy disabled}
```

# Your environment (if applicable)

* Version used: alphaTab v1.0.1 (, build 2)
* Platform used: JavaScript
* Rendering engine used: SVG (default)
* Browser Name and Version: Chrome Version 87.0.4280.67
* Operating System and version (desktop or mobile): Darwin Kernel Version 16.7.0
* Link to your project: https://sugizo.pythonanywhere.com/music/notation/form_tablature_in_notation

thanks

**Comment by Danielku15** [comment, marked as answer]
Hiding UI elements is not part of alphaTex. alphaTex defines the music piece and there dynamics and tempo are an essential part of it. 

Hiding UI elements is part of the settings you pass on to alphaTab for rendering, like the layout: 
https://www.alphatab.net/docs/reference/settings/notation/elements

In your case you would likely do something like: 

```js
const api = new alphaTab.AlphaTabApi(document.querySelector('#alphaTab'), {
    notation: {
        elements: {
            effectTempo: false,
            effectDynamics: false
        }
    }
});

```

**Reply by sugizo** [reply]
nice to have that alphatex can disable it by throw some value (e.g. none, false, disable, disabled, hide, hidden, 0)
like time signature can be disabled via
`\ts 0 0`

**use case**
while teach or learn arpeggio, scale or chord the lecture can show the tablature without tempo or dynamics

**Reply by Danielku15** [reply]
I don't think I will duplicate all these settings also into alphaTex. If you have such a need for your platform, I recommend that you store the settings to use for rendering along the alphaTex code and pass it along during initializations.
